README.md

```diff
@@ -100,149 +110,39 @@ depends:
 
 ### Add controls
 
-Since your profile depends on the resource pack, you can use those resources in your own profile:
-
-```ruby
-control 'azure-1' do
-  impact 1.0
-  title 'Checks that the machine was built from the correct image'
-
-  describe azure_virtual_machine(name: 'example-01', resource_group: 'MyResourceGroup') do
-    its('sku') { should eq '16.04.0-LTS' }
-    its('publisher') { should ieq 'Canonical' }
-    its('offer') { should ieq 'UbuntuServer' }
-  end
-end
-```
-
-### Available Resources
-
-- `azure_resource_group` - This resource reads information about the resources in the specified resource group
+Since your profile depends on the resource pack, you can use those resources in your own profile. In this example a virtual machine in the specified resource group is being tested.
 
-| Resource Name | Resources | Description |
-|---------------|-----------|-------------|
-| azure_rg | location | Where the item has been deployed |
-| | total | The total number of resources in the resource group |
-| | nic_count | Helper method to return the number of Network Interface Cards (NIC) that exist |
-| | vm_count | Helper method to return the number of Virtual Machines(VM) that exist |
-| | vnet_count | Helper method to return the number of Virtual Networks (VNET) that exist |
-| | nsg_count | Helper method to return the number of Network Security Groups (NSG) that exist |
-| | sa_count | Helper method to return the number of Storage Accounts (SA) that exist |
-| | public_ip_count | Helper method to return the number of Public IP Addresses that exist |
-| | managed_disk_image_count | Helper method to return the number of Managed Disk Images that exist |
-| | managed_disk_count | Helper method to return the number of Managed Disks that exist |
-| | contains | Used to determine if a specific item exists |
+Using the https://resources.azure.com portal the virtual machine has the following attributes.
 
-This resource also has a FilterTable which means that it is possible to check for items that do not yet have a helper method.
-
-#### Test for the number of storage accounts
-
-```ruby
-control 'azure-1' do
-  impact 1.0
-  title 'Checks that there is only one storage account in the resource group'
-
-  describe azure_resource_group(name: 'MyResourceGroup').where { type == 'Microsoft.Storage/storageAccounts' }.entries do
-    its('count') { should eq 1 }
-  end
-end
-```
+![Virtual Machine Attributes](images/linux_internal_vm_resource.png)
```

Comment:
I'm not sure how to handle this - linking to a screenshot of a JSON dump might not be the best way.  I might suggest that we put more examples, have them come first, and then perhaps link to Azure's documentation, or failing that, if the screenshot is the only way, that will have to do.  @kagarmoe ?

Reply:
This is a screenshot of what is displayed by https://resources.azure.com and is there to show people what they can expect to see when thet are interogating Azure Resources.
I can change it so that it shows more of the website so it is more explicit, but this is not a JSON object that we created.